lib/lotus/routing/routes_inspector.rb

```diff
@@ -32,6 +43,7 @@ def initialize(routes)
       #
       # @param formatter [String] the optional formatter for a route
       # @param base_path [String] the base path of a nested route
+      # @param header [Boolean] display header test (default false)
```

Comment:
Please remove this boolean. They are always a code smell. If we think that the header line is useful (as I think it is), there is no need of this flag. All the inspections will have this needed information by default.